react/Tooltip/index.jsx

```diff
@@ -0,0 +1,18 @@
+import { withStyles } from '@material-ui/core/styles'
+import { Tooltip } from '@material-ui/core'
+
+const styles = {
```

Comment:
Can you add a reason why this had to be managed in CSS-in-JS ?

Reply:
Initially it was because the component was used in cozy-notes, but now that we add it in UI I think we could put this css in the MUI theme no?

Reply:
done